azure/scenarios/vm/vm/run.go

```diff
@@ -0,0 +1,24 @@
+package vm
+
+import (
+	"github.com/DataDog/test-infra-definitions/azure"
+	"github.com/DataDog/test-infra-definitions/azure/compute"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+)
+
+func Run(ctx *pulumi.Context) error {
+	env, err := azure.AzureEnvironment(ctx)
+	if err != nil {
+		return err
+	}
+
+	_, publicIP, nw, adminPassword, err := compute.NewWindowsInstance(env, "vm", compute.WindowsLatestURN(), env.DefaultInstanceType(), nil, nil)
+	if err != nil {
+		return err
+	}
+
+	ctx.Export("private-instance-ip", nw.IpConfigurations.Index(pulumi.Int(0)).PrivateIPAddress())
+	ctx.Export("public-instance-ip", publicIP.IpAddress)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Why do we need private and public address ?

Reply:
I have a bastion available in this network as well. Only SSH/RDP is allowed from external. If you need any other kind of interaction you will need to go through the bastion (and private ip)